src/plotting_static.jl

```diff
@@ -160,27 +160,26 @@ end
 """
     plot_1D_solutions(res::Result; 
                         x::String, y::String, 
-                        x_scale=1.0, y_scale=1.0, 
                         marker="o",xscale="linear",yscale="linear"
                         ,plot_only=["physical"],
-                        marker_classification="stable",filename=nothing)
+                        marker_classification="stable",filename=nothing, kwargs...)
 
 Make a 1D plot of a `Result` object.    
 
 Keyword arguments
 - `x`, `y`: Expressions to plot on as independent/dependent variables (parsed into Symbolics.jl).
-- `x_scale`, `y_scale`: Factors to multiply the shown axis ticks with.
 - `marker`: The point marker to use.
 - `xscale`, `yscale`: scale for x/y dimensions (e.g. "linear" or "log")
 - `plot_only`: a list of strings corresponding to the solution classes of `Result`. Only solutions which belong to the listed classes are plotted.
 - `marker_classification`: A class of the solutions (created by `classify_solutions!`) which is distinguished with different markers. Entering an inequality creates a new class "custom_class".
 - `filename`: if different from `nothing`, plotted data and parameter values are exported to `./filename.jld2`.
 - `col_length`: number of elements per legend column. By default 10.
```

Comment:
this argument has now been removed, although I suggest not doing so. See comment below